runners/flink/build.gradle

```diff
@@ -72,6 +72,7 @@ dependencies {
   shadowTest library.java.junit
   shadowTest library.java.mockito_core
   shadowTest library.java.google_api_services_bigquery
+  shadowTest project(path: ":sdks:java:io:google-cloud-platform", configuration: "shadow")
```

Comment:
Why is it `"shadow"` and not `"shadowTest"`? I must admit I don't have much experience with Gradle.

Reply:
`shadow` is correct.

I wrote this up to help people when choosing configurations.
https://github.com/apache/beam/blob/5b99fe2ea44e1a0d8a6f802ca757d1a1cfab91cc/build_rules.gradle#L150